common/static/common/js/spec/discussion/view/new_post_view_spec.js

```diff
@@ -94,7 +94,8 @@
                 this.view.render();
                 expectedGroupId = null;
                 DiscussionSpecHelper.makeAjaxSpy(function(params) {
-                    return expect(params.data.group_id).toEqual(expectedGroupId);
+                    expect(params.data.group_id).toEqual(expectedGroupId);
+                    self.view.$(".forum-new-post-form").removeAttr("disabled");
```

Comment:
This seems like a weird side effect for a method called `makeAjaxSpy`. Can you put this somewhere else?